ucd-list.txt

```diff
@@ -133,6 +133,9 @@ P | meta.code.mime                    |  MIME type
 P | meta.code.multip                  |  Multiplicity or binarity flag
 P | meta.code.qual                    |  Quality, precision, reliability flag or code
 P | meta.code.status                  |  Status code (e.g.: status of a proposal/observation)
+Q | meta.coverage                     |  A coverage (spatial, temporal, spectral, etc)
+with any combination of physical axis (such as moc, tmoc, stmoc, s_region).
```

Comment:
I think s_region should be left out here; it's not a datatype.  You could write "as in obscore's s_region", but I don't think that would help.  Also, it'd be "ax*e*s" here.

Reply:
Ok , thanks .